test/Abp.Quartz.Tests/QuartzTests.cs

```diff
@@ -33,7 +32,7 @@ private async Task ScheduleJobs()
                 {
                     trigger.WithIdentity("HelloJobTrigger")
                            .WithDescription("HelloJobTriggerDescription")
-                           .WithSimpleSchedule(schedule => schedule.WithRepeatCount(5).WithInterval(TimeSpan.FromSeconds(1)))
+                           .WithSimpleSchedule(schedule => schedule.WithRepeatCount(50).WithInterval(TimeSpan.FromSeconds(1)))
```

Comment:
Why change 5 to 50?

Reply:
The trigger will be removed from the scheduler after those 5 times. By the time I get to the Reschedule method, the trigger will no longer be there and there is nothing to reachedule. I doesn't have to run 50 timea exactly, it just makes the testing safe from failing.